spark/src/main/java/org/apache/iceberg/spark/SparkUtil.java

```diff
@@ -100,4 +121,56 @@ public static void validatePartitionTransforms(PartitionSpec spec) {
       }
     }
   }
+
+  /**
+   * Responsible for checking if the table schema has a timestamp without timezone column
+   * @param schema table schema to check if it contains a timestamp without timezone column
+   * @return boolean indicating if the schema passed in has a timestamp field without a timezone
+   */
+  public static boolean hasTimestampWithoutZone(Schema schema) {
+    return TypeUtil.find(schema, t ->
+            t.typeId().equals(Type.TypeID.TIMESTAMP) && !((Types.TimestampType) t).shouldAdjustToUTC()
```

Comment:
could just check against TimestampType.withoutZone() which returns the singleton instance

Reply:
good point, changed the code